diff --git a/cpp/BUILD.bazel b/cpp/BUILD.bazel
index f1bfd36..33b214d 100644
--- a/cpp/BUILD.bazel
+++ b/cpp/BUILD.bazel
@@ -19,6 +19,6 @@ proto_plugin(
         "{basename}.grpc.pb.cc",
         "{basename}_mock.grpc.pb.h",
     ],
-    tool = "@com_github_grpc_grpc//:grpc_cpp_plugin",
+    tool = "@com_github_grpc_grpc//src/compiler:grpc_cpp_plugin",
     visibility = ["//visibility:public"],
 )
diff --git a/deps.bzl b/deps.bzl
index a4267c0..d3aff74 100644
--- a/deps.bzl
+++ b/deps.bzl
@@ -324,8 +324,8 @@ def io_grpc_grpc_java(**kwargs):
     """grpc java plugin and jars
     """
     name = "io_grpc_grpc_java"
-    ref = get_ref(name, "3c24dc6fe1b8f3e5c89b919c38a4eefe216397d3", kwargs)  # v1.19.0 and changes up to PR #5456
-    sha256 = get_sha256(name, "1eeb136874a58a0a311a0701016aced96919f501ced0372013eb1708724ab046", kwargs)
+    ref = get_ref(name, "e7d7c5bde4caf606f02b2575d5f62e80ad5135b0", kwargs)  # head @ Jan 20 2020
+    sha256 = get_sha256(name, "6674f477e714d4d40f8c40d46df1b9dc7e192d3e1d9ee8d288135dc6da995d70", kwargs)
     github_archive(name, "grpc", "grpc-java", ref, sha256)
 
 
diff --git a/python/BUILD.bazel b/python/BUILD.bazel
index 67ab2cd..e6ea86f 100644
--- a/python/BUILD.bazel
+++ b/python/BUILD.bazel
@@ -9,7 +9,7 @@ proto_plugin(
 proto_plugin(
     name = "grpc_python",
     outputs = ["_pb2_grpc.py"],
-    tool = "@com_github_grpc_grpc//:grpc_python_plugin",
+    tool = "@com_github_grpc_grpc//src/compiler:grpc_python_plugin",
     transitivity = {
         "google/protobuf": "exclude",
     },
